lib/galaxy/tools/relabel_from_file.xml

```diff
@@ -137,23 +137,75 @@
     </tests>
     <help><![CDATA[
 
+========
+Synopsis
+========
+
+This tool changes names of datasets within a collection using identifiers from a supplied file. 
```

Comment:
```suggestion
This tool changes identifiers of datasets within a collection using identifiers from a supplied file. 
```
It doesn't change the dataset name. If you unhide the collection you see the dataset names.